storage/src/mmr/bitmap.rs

```diff
@@ -53,9 +78,9 @@ impl<H: CHasher> Bitmap<H> {
     }
 
     /// Return the last chunk of the bitmap as a slice.
-    fn last_chunk(&self) -> &[u8] {
+    fn last_chunk_as_digest(&self) -> H::Digest {
```

Comment:
nit: I think the `H::Digest` obviates `as_digest`? Same for `get_chunk_as_digest`.

imho it is ok to have last_chunk return digest and last_chunk_mut return &[u8]

Reply:
done